e2e/bootstrap.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd $(dirname $0)
+
+# Clean up potential data from previous runs,
+# in order to start with a clean state.
+docker-compose down -v
+rm -rf ./matrix_data/homeserver.db* ../sqlite.db
+
+
+# Start Polkadot and Matrix and wait until they are up.
+docker-compose up -d
+source wait_until.sh 'curl -s "localhost:8008"'
+source wait_until.sh 'curl -s "localhost:9933"'
```

Comment:
This should technically be 127.0.0.1 instead of localhost

Reply:
Ok, updated.